### DISCUSSION TITLE
How to receive DEBUG message

### DISCUSSION THREAD

**Discussion by leo-lu-niz** [COVESA/dlt-daemon#532] [Q&A]
Hi Team,

I am using dlt-receive, but I have no idea why I could not receive DEBUG message?

I could receive other levels， so I guess it is not the connection problem.

Any help or guidance would be awesome.. thanks in advance.

Leo

**Comment by minminlittleshrimp** [comment]
Hello @leo-lu-niz 
I would like to know more info from your side.
Do you set dlt.conf fields corectly? What app you want to get debug msgs? Could you try with dlt-example-user and receive the same issue?
